contracts/governance/TokenholderGovernor.sol

```diff
@@ -0,0 +1,143 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.9;
+
+import "./TokenholderGovernorVotes.sol";
+import "../token/T.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/governance/Governor.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorVotes.sol";
+import "@openzeppelin/contracts/governance/extensions/GovernorTimelockControl.sol";
+
+contract TokenholderGovernor is
+    AccessControl,
+    Governor,
+    GovernorCountingSimple,
+    TokenholderGovernorVotes,
+    GovernorTimelockControl
+{
```

Comment:
Maybe possible to extract some logic from this and `StakerGovernor` to some other abstract contract?